javascript/new-src/Gruntfile.js

```diff
@@ -124,7 +119,7 @@ module.exports = function(grunt) {
     cssmin: {
       build: {
         files: [{
-          src: 'css/style.css',
+          src: targets.css,
```

Comment:
It still works, the way they document is just a simpler way of writing it. The dest/src properties get handled by grunt if I recall correctly